morevna-ep4.msrt

```diff
@@ -46,17 +46,17 @@
 10
 00:01:09,600 --> 00:01:12,460
 [rus] И я приняла её...
-[eng] And I'd accepted it...
+[eng] And I accepted it...
```

Comment:
It doesn't feel right to mix tenses since these lines form a single sentence, so I opted to use the simple past tense throughout. If Marya still accepts it, and still is it (in the following line), then this is the better tense to use. If she no longer accepts it or is it, then it would be better to use a compound past tense for all of these lines (ie. It had become my only way...). At least that's how I would do it.